tox.ini

```diff
@@ -1,11 +1,12 @@
 [tox]
-env_list = py{310,311,312},notebooks
+env_list = py{310,311,312,313},notebooks
```

Comment:
Should we run notebooks etc. with latest supported python version? Or even with all python versions?